db/migrations/20220115171101_init.up.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE public.users (
+    id SERIAL PRIMARY KEY,
+    email character varying UNIQUE,
+    login character varying NOT NULL UNIQUE,
+    password character varying NOT NULL,
+    created_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    updated_at timestamp with time zone
+);
+
+--bun:split
+
+CREATE TABLE public.invoices (
+    id SERIAL PRIMARY KEY,
+    type character varying,
+    user_id bigint,
+    amount bigint,
+    memo character varying,
+    description_hash character varying,
+    payment_request character varying,
+    destination_pubkey_hex character varying NOT NULL,
+    r_hash character varying,
+    preimage character varying,
+    internal boolean,
+    state character varying DEFAULT 'initialized'::character varying,
+    error_message character varying,
+    add_index bigint,
+    created_at timestamp with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    expires_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    settled_at timestamp with time zone,
+    CONSTRAINT fk_user
+        FOREIGN KEY(user_id) 
+        REFERENCES users(id)
+        ON DELETE CASCADE
```

Comment:
just to double be sure: this deletes the invoice of a user when the user is deleted. but NOT the other way around, correct?

Reply:
Yes, child table entry deleted when parent table entry is deleted